lib/components/navbar-brand.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+     <b-link v-if="isLink"
+             class="navbar-brand"
+             v-bind="linkProps"
+             @click="$emit('click', $event)">
+        <slot></slot>
+    </b-link>
+    <div v-else :is="tag" class="navbar-brand">
+        <slot></slot>
+    </div>
```

Comment:
Might make slightly more sense just using the `<component>` tag here. Just shows it off clearly as a dynamic tag.